docker-compose.prod.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  app:
+    image: app-api:latest
+    networks:
+      - caddy_public
+    volumes:
+      - caddy_data:/data
+      - caddy_config:/config
+      - runtime:/app/app/runtime
+    environment:
+      SERVER_NAME: ":80"
+    deploy:
+      restart_policy:
+        condition: on-failure
+        delay: 5s
+        max_attempts: 3
+        window: 120s
+      labels:
+        caddy: app-api.example.com
+        caddy.reverse_proxy: "{{upstreams 80}}"
```

Comment:
What about https?

Reply:
No need for it if you're behind a reverse proxy.

Reply:
Made it 80 + configurable.